R/label-dollar.R

```diff
@@ -102,6 +126,18 @@ dollar <- function(x, accuracy = NULL, scale = 1, prefix = "$",
   negative <- !is.na(x) & x < 0
   x <- abs(x)
 
+  if (!is.null(rescale_large)) {
+    if (!(is.integer(rescale_large) && all(rescale_large > 0))) {
+      stop("`rescale_large` must be positive integers.", call. = FALSE)
+    }
+
+    rescale <- rescale_by_suffix(x * scale, breaks = c(0, 10^rescale_large))
```

Comment:
I think `rescale_by_suffix()` should probably handle the `10^`

Reply:
Glad you brought this up! It reminded me... 

I meant to ask if I should refactor `label_bytes(units = "auto_si|auto_binary")` to also use `rescale_by_suffix()`? If so, then we don't want `rescale_by_suffix()` to handle the `10^`.

Reply:
Yeah, if you think it fits.